css/site.styl

```diff
@@ -31,49 +31,111 @@ normalize()
 // GRID
 
 $cols = 12
-$col = 90px
-$gutter = 16px
+$gutter = 24px
 
 .container
   margin: 0 auto
-  width: ($col * $cols)+($gutter * $cols)
+  width: 100%
 
 .row
-  clearfix()
+  box-sizing: border-box
+  display: flex
+  flex: 0 1 auto
+  flex-direction: row
+  flex-wrap: wrap
+  &.reverse
+    flex-direction: row-reverse
+
+// FLEX POSITIONING
+
+.justify-start
+  justify-content: flex-start
+
+.justify-center
+  justify-content: center
+
+.justify-end
+  justify-content: flex-end
+
+.align-start
+  align-items: flex-start
+
+.align-center
+  align-items: center
+
+.align-end
+  align-items: flex-end
+
+.align-content-start
+  align-content: flex-start
+
+.align-content-center
+  align-content: center
+
+.align-content-end
+  align-content: flex-end
+
+// COLUMN SIZES & OFFSETS
 
 .col
-  float: left
-  margin-left: ($gutter/2)
-  margin-right: ($gutter/2)
-  min-height: 1px
+  flex: 1 1 auto
+  box-sizing: border-box
+  max-width: 100%
+  padding: 0 ($gutter / 2)
+
```

Comment:
No I don't like the pattern still and I already needed to remove it in this feature branch for it to look as it should. This isn't mobile first tho so maybe works better like that